cmd/operator/main.go

```diff
@@ -109,8 +109,8 @@ func main() {
 	logger.SetLevel(loggingLevel)
 
 	if !isatty.IsTerminal(os.Stdout.Fd()) {
-		logger.Formatter = stackdriverFormatter.NewFormatter(
-			stackdriverFormatter.WithService(os.Getenv("K_SERVICE")),
+		logger.Formatter = sdlog.NewFormatter(
+			sdlog.WithService(os.Getenv("K_SERVICE")),
```

Comment:
Can you add the fallback as discussed too

Reply:
Ok, I'll do that here too.